2022-01-05 13:17:33.593227 HTTPConnectionPool(host='connect-srvc.monit.tech', port=80): Max retries exceeded with url: /api/SetDeviceHeartBeat?hubid=29&&servertime=2022-01-05T13:16:54&&lat=0&&long=0&&alt=0.0&&speed=0&&cabin=false&&front=false&&left=false&&right=false (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x73d649d0>: Failed to establish a new connection: [Errno 110] Connection timed out'))